test/test.js

```diff
@@ -3,64 +3,76 @@ var assert = require('assert');
 var getWindow = require('../');
 var getDocument = require('get-document');
 
+var inNode = require('detect-node');
+
+// Zuul is configured to pass on to Browserify a configuration to exclude the
+// following module, 'jsdom' because it is only for testing in node. Browsers
+// have real dom
+var jsdom = inNode ? require('jsdom') : null;
+
+var doc = inNode ? jsdom.jsdom() : document;
```

Comment:
Instead of using a global `document` throughout (we don't have that in node), we assign to `doc` either a JavaScript DOM document, in the case of node, or the global `document`, in the case of a browser.